PR bots failsAPI/gen diff step

https://github.com/xamarin/xamarin-macios/pull/10220

🔥 Failed to compare API and create generator diff 🔥
    Failed to build src/
    Search for Comparing API & creating generator diff in the log to view the complete log.

Logs shows

> make: *** [/Users/builder/jenkins/workspace/xamarin-macios-pr-builder/tools/comparison/build/maccatalyst/maccatalyst/generated_sources] Error 1

It might just affect the HEAD commit, i.e. it will fix itself with another commit.

--- a/tools/compare-commits.sh
+++ b/tools/compare-commits.sh
@@ -180,7 +180,7 @@ mkdir -p "$OUTPUT_DIR/project-files"
 ln -s git "$OUTPUT_DIR/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/Current"
 ln -s git "$OUTPUT_DIR/_mac-build/Library/Frameworks/Xamarin.Mac.framework/Versions/Current"
 
-for dir in 2.1 Xamarin.iOS Xamarin.TVOS Xamarin.WatchOS; do
+for dir in 2.1 Xamarin.iOS Xamarin.TVOS Xamarin.WatchOS Xamarin.MacCatalyst; do
 	$CP -R "$ROOT_DIR/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono/$dir" "$OUTPUT_DIR/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono"
 done
 

